Dear/Dear/Controllers/NursePrescriptionViewController.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  NursePrescriptionView.swift
+//  Dear
+//
+//  Created by Jeon Jimin on 2022/07/28.
+//
+
+import UIKit
+
+class NursePrescriptionViewController: UIViewController {
+    
+    //TODO: 서버 데이터 받아와서 연결하기
+    let letterSampleData = Letter.sampleData[0]
+
+    @IBOutlet weak var dateLabel: UILabel!
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var contentLabel: UILabel!
+    
+    @IBOutlet weak var depressionView: UIView!
+    @IBOutlet weak var sadnessView: UIView!
+    @IBOutlet weak var angerView: UIView!
+    @IBOutlet weak var calmView: UIView!
+    @IBOutlet weak var joyfulView: UIView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        dateLabel.text = letterSampleData.date
+        nameLabel.text = letterSampleData.letterTo
+        contentLabel.text = letterSampleData.letterContent
+        
+        setMedicine()
+    }
+    
+    private func setMedicine() {
+        depressionView.isHidden = letterSampleData.depression ? false : true
+        sadnessView.isHidden = letterSampleData.sadness ? false : true
+        angerView.isHidden = letterSampleData.anger ? false : true
+        calmView.isHidden = letterSampleData.calm ? false : true
+        joyfulView.isHidden = letterSampleData.joyful ? false : true
```

Comment:
```suggestion
        depressionView.isHidden = !letterSampleData.depression
        sadnessView.isHidden = !letterSampleData.sadness
        angerView.isHidden = !letterSampleData.anger
        calmView.isHidden = !letterSampleData.calm
        joyfulView.isHidden = !letterSampleData.joyful
```

이렇게하면 조금 더 심플하지 않을까해요 ~

Reply:
헉 제가 삼항연산자까지는 생각했는데 더 간단한 방법이있었군요!! 감사합니다~!!